src/modules/thesaurus/src/main/java/org/geonetwork/thesaurus/controller/ThesaurusController.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * (c) 2003 Open Source Geospatial Foundation - all rights reserved
```

Comment:
If this is porting work from GN4 then the date of the original should be preserved.
If it is a new file then 2025 (date of file creation) would be expected.